test/test_helper.exs

```diff
@@ -12,6 +12,17 @@ unless Docker.ready? do
   exit({:shutdown, 1})
 end
 
+{output, _} = System.cmd("sha1sum", ["--version"])
```

Comment:
i like that you try to help the user to find out (s)he didn't install `sha1sum`, but (at least for me on macos) i get the following error in that case:

```
mix test
** (ErlangError) erlang error: :enoent
    (elixir) lib/system.ex:564: System.cmd("sha1sum", ["--version"], [])
    test/test_helper.exs:15: (file)
    (elixir) lib/code.ex:370: Code.require_file/2
    (elixir) lib/enum.ex:645: Enum."-each/2-lists^foreach/1-0-"/2
    (elixir) lib/enum.ex:645: Enum.each/2
```

The code seems to fail before your nice error message gets rendered.